config/defaults.mk

```diff
@@ -301,45 +300,24 @@ PUMI_OPT = -I$(PUMI_DIR)/include
 PUMI_LIB = -L$(PUMI_DIR)/lib -lpumi -lcrv -lma -lmds -lapf -lpcu -lgmi -lparma\
    -llion -lmth -lapf_zoltan -lspr
 
-# CUDA library configuration
-ifeq ($(MFEM_USE_CUDA),YES)
-   ifndef CUDA_DIR
-      CUDA_DIR := /usr/local/cuda
-   endif
-   CUDA_OPT := -I$(CUDA_DIR)/include
-   CUDA_LIB := -L$(CUDA_DIR)/lib64 -lcuda -lcudart
-endif
-
-# MPI library configuration
-ifeq ($(MFEM_USE_MPI),YES)
-   ifndef MPI_DIR
-      MPI_DIR := $(HOME)/local/openmpi/3.0.0
-   endif
-   MPI_OPT := -I$(MPI_DIR)/include
-   MPI_LIB := -L$(MPI_DIR)/lib -lmpi
-endif
+# CUDA library configuration. Since we compile and link with nvcc (when CUDA is
+# enabled) we only need to explicitly link with the CUDA driver, libcuda.*,
+# which is usually in a system path.
```

Comment:
Both `CUDA_DIR` and `MPI_DIR` are currently mentioned in the `INSTALL` file. If we don't need them (which will be great) then this PR should also update `INSTALL`

Reply:
Good point.